src/App.js

```diff
@@ -22,12 +26,22 @@ export class App extends Component {
                   id="search-query"
                   className="input"
                   placeholder="Type search word"
+                  value={query}
+                  onChange={(event) => {
+                    this.setState({
+                      query: event.target.value,
+                    });
+                  }}
                 />
               </div>
             </div>
           </div>
 
-          <MoviesList movies={moviesFromServer} />
+          <MoviesList movies={
+            moviesFromServer.filter(movie => movie.title
+              .toLowerCase().includes(query.toLowerCase()))
```

Comment:
Zhenya is right. You should search not only in the movie's title but also in its description (as the task says):

> 2. Filter the movies using state.query by searching it in the movie.title or movie.description